cxx-checks/src/main/java/org/sonar/cxx/checks/FunctionComplexityCheck.java

```diff
@@ -46,7 +47,8 @@
   offset = "10min",
   effortToFixDescription = "per complexity point above the threshold")
 public class FunctionComplexityCheck extends SquidCheck<Grammar> {
-  private static final int DEFAULT_MAX = 10;
+
+  private static final int DEFAULT_MAX = 8;
```

Comment:
My idea was to set @TemplateRule for this.
Comparison is always
   create message if CC > limit
So it is no problem to set different ascending values.

> Am 30.06.2015 um 21:18 schrieb Jorge Costa <notifications@github.com>:
> 
> In cxx-checks/src/main/java/org/sonar/cxx/checks/FunctionComplexityCheck.java:
> 
> > @@ -46,7 +47,8 @@
> >    offset = "10min",
> >    effortToFixDescription = "per complexity point above the threshold")
> >  public class FunctionComplexityCheck extends SquidCheck<Grammar> {
> > -  private static final int DEFAULT_MAX = 10;
> > +
> > +  private static final int DEFAULT_MAX = 8;
> But everything bellow 10 imo should not raise any issues
> …
> —
> Reply to this email directly or view it on GitHub.
>